dojo/tools/factory.py

```diff
@@ -65,10 +65,12 @@
 from dojo.tools.hadolint.parser import HadolintParser
 from dojo.tools import SCAN_SONARQUBE_API
 from dojo.tools.aqua.parser import AquaJSONParser
+from dojo.tools.h1.parser import HackerOneJSONParser
 from dojo.tools.xanitizer.parser import XanitizerXMLParser
 from dojo.tools.trivy.parser import TrivyParser
 
 
+
```

Comment:
Will not pass flake8 I would assume. Please make sure you run flake8 tests and fix warnings (and check the box in the PR description when done).